scripts/lua/modules/lua_utils.lua

```diff
@@ -980,6 +980,18 @@ function isValidPoolMember(member)
   return false
 end
 
+function host2member(ip, vlan)
```

Comment:
see function hostinfo2hostkey, I think you can reuse it, e.g., hostinfo2hostkey({host="",vlan=0}) perhaps you can extend it with subnet